### PR TITLE
[IOS-437]: Group chat feature

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Toshi/Controllers/Browse/BrowseViewController.swift [hunk 1]
```diff
@@ -126,6 +126,32 @@ class BrowseViewController: SearchableCollectionController {
...
+            guard let info = notification.userInfo else { return }
+            let duration = info[UIKeyboardAnimationDurationUserInfoKey] as? TimeInterval ?? 0.2
+            let curveRaw = info[UIKeyboardAnimationCurveUserInfoKey] as? UInt ?? 0
+            let curve = UIViewAnimationOptions(rawValue: curveRaw)
+
+            UIView.animate(withDuration: duration, delay: 0, options: UIViewAnimationOptions(rawValue: UInt(curve.rawValue)), animations: {
+                self?.searchResultView.contentInset.bottom = 0
+                self?.collectionView.contentInset.bottom = self?.bottomInset ?? 0.0
+            })
+        }
```
Comment (designatednerd) [inline Toshi/Controllers/Browse/BrowseViewController.swift]: is this the stuff that's supposed to be replaced or am I thinking of something else

Comment (yuliaveres) [reply Toshi/Controllers/Browse/BrowseViewController.swift]: replaced? i think it is one of the ui improvements prs. merged into here for some reason. see in blame

**Comment by yuliaveres** [comment]
Remove the odd BrowseViewController UI change

File: Toshi/Models/SofaWrappers/SofaWrapper.swift [hunk 2]
```diff
@@ -53,6 +55,8 @@ enum SofaType: String {
             self = .paymentRequest
         } else if sofa.hasPrefix(SofaType.payment.rawValue) {
             self = .payment
+        } else if sofa.hasPrefix(SofaType.payment.rawValue) {
+            self = .status
```
Comment (designatednerd) [inline Toshi/Models/SofaWrappers/SofaWrapper.swift]: ^^ looks like think this slipped through - you can fix after merge if you want though

**Pull request merged**